### DISCUSSION TITLE
Question on intergration with docker

### DISCUSSION THREAD

**Discussion by farzadha2** [utmstack/UTMStack#513] [Q&A] [answered]
Hi
Currently was looking for the integration for docker to monitor the containers, i see in the Docs that it supports it but was looking for the wiki didnt find it if someone could shed some light
https://utmstack.com/wp-content/uploads/2022/06/utmstack.pdf
THanks

**Comment by c3s4rfred** [comment]
Hi @farzadha2, we will check our integrations and give you feedback, best regards

**Comment by c3s4rfred** [comment, marked as answer]
Hi, @farzadha2 that document is outdated, please use this https://docs.utmstack.com/
